CHANGELOG.md

```diff
@@ -15,6 +15,10 @@
 * [#4249](https://github.com/bbatsov/rubocop/issues/4249): Handle multiple assignment in `Rails/RelativeDateConstant`. ([@bbatsov][])
 * [#4250](https://github.com/bbatsov/rubocop/issues/4250): Improve a bit the Ruby code detection config. ([@bbatsov][])
 
+### New features
```

Comment:
Whoops, the section didn't exist when I first added it. I have rebased and removed the duplicate section.